src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -246,20 +255,6 @@ export function sipFormSorter(formA, formB) {
 export const isFormAuthorizable = formConfig =>
   !!formConfig && !!formConfig.authorize;
 
-export const getFormAuthorizationState = (formConfig, state) =>
```

Comment:
This was only used in a forms library component, so I moved the whole function into the forms library.